### DISCUSSION TITLE
VSCode plugin does not work in three ways: signin, feedback, and chat

### DISCUSSION THREAD

**Discussion by ekernf01** [community/community#114737] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi! Excited to try out copilot for the first time! I'm using VSCode on Ubuntu 20.04. I have an email showing that my free trial has started. I followed the [install instructions for vscode](https://marketplace.visualstudio.com/items?itemName=GitHub.copilot) and the extension was installed. Unfortunately I'm having three problems getting it up and running in the IDE. 

1. For a long time the sign-in button had no effect. I was eventually able to sign in, but there was never an error message, and I do not know what determines sign-in success.
2. The chat functionality is unresponsive. For instance if I type "help" and click the button in the bottom right part of the chat window (button has a "send" icon similar to a garden-variety messaging app), nothing happens. 
3. The feedback functionality is unresponsive. When I click the button with a cartoon snowman with a speech bubble (hover text is "send chat feedback"), I get an error `command 'github.copilot.interactiveSession.feedback' not found`. 

I did some reading on the support forums but I'm pretty stuck. Do you have any recommendations for how to get started? Thanks!

Edit: here's more of what I tried. 

- Tried the solutions at https://github.com/orgs/community/discussions/84914 . I was able to confirm my subscription, but the command "Github copilot: sign out" is not available to me. I tried reloading the window, but this brings me back to sign-in, which as I said above works only intermittently. Tried updating copilot but I just installed and I'm on the latest version already. I tried making a personal access token, and I made the token successfully, but VS Code does not offer me a command "GitHub Copilot: Personal Access Token". 
- Looked at this thread https://github.com/orgs/community/discussions/77195 , but I don't know what "-insiders" is and I never installed something by that name. Googling, it looks like a windows version of vscode, and I'm on ubuntu. It says "DO NOT USE THE OLD VERSION OF COPILOT CHAT" but I just today installed the currently recommended copilot extension.

**Comment by Manjunathhs-0003** [comment]
Open the command palette (Ctrl + Shift + P)
Run Disable All Installed Extensions
Then run Enable All Extensions.
Restart Visual Studio Code
You can make sure again that the extensions are enabled.

**Reply by ekernf01** [reply]
I tried this, and I verified that the extensions are enabled, but the behavior is the same so far. Sign-in button has no effect.

**Comment by Manjunathhs-0003** [comment]
ok i found this in another github discussion, thought might help
https://github.com/VSCodium/vscodium/issues/1546#issuecomment-1793619431

**Reply by ekernf01** [reply]
I'm sorry, I'm not willing to try this solution. Compared to my situation, this thread describes a different problem with a different editor, and it involves poking around with files that I am sure VSCode does not intend me to poke.

**Comment by Manjunathhs-0003** [comment]
you can try one or the other, might work
https://github.com/orgs/community/discussions/6785
